DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1516,7 +1517,6 @@ function MarkBingoFailedSpecial()
         MarkBingoAsFailed("ChangeClothes");
         break;
     case "10_PARIS_CATACOMBS":
-    case "11_PARIS_EVERETT":
```

Comment:
removing this is pretty interesting, because this backtracking is only enabled during bingo modes